### DISCUSSION TITLE
Is this mod still in development?

### DISCUSSION THREAD

**Discussion by thedocruby** [VideoGameSmash12/HotbarsPlus#7] [Q&A] [answered]
I have been looking for a mod like this for ages now, and the only one i found was abandoned (more toolbars).

Then i found a link to this mod on some obscure freedom server forum and it seemed even more organized and thought through. However, the last status update is from four months ago. Is this mod still in development? If so, I will be using it in my upcoming YouTube channel, and it may get some publicity for that. If not, i will be forced to make my own or move on without it. I understand if the project was forgotten due to lack of interest but i am here to tell you that i will get much use out of this mod if it were implemented.

Also the readme says something about a keybind for changing the hotbar page without opening the inventory, which is absolutely brilliant, but I cant find where it says what the keybind is. Where would i find that, or how do I go about setting the keybind? (I haven't actually loaded the mod yet because I'm out of town so its probably just in the config)

**Comment by thedocruby** [comment]
Also note that I am looking for a mod like this to help organize my redstone development process, and when I include all of the colored blocks (which i am using a separate hotbar for each color, the block type has meaning and the color is for labeling separate parts of the circuit) plus 6 or more hotbars for redstone components, this quickly becomes too much for the vanilla saved toolbar system. This mod you have developed would make things incredibly simple, and that is something that is much appreciated when you are focusing on developing highly complex redstone contraptions

**Comment by VideoGameSmash12** [comment, marked as answer]
Apologies for the late response! This mod was in a lengthy development hiatus as I have been busy with other projects, but I plan to continue development in the near future. The keybinds you are looking for are `[` (for previous page) and `]` (for next page).

Hope this helps!
